components/brave_wallet/browser/solana_transaction.cc

```diff
@@ -18,6 +18,118 @@
 
 namespace brave_wallet {
 
+namespace {
+
+constexpr char kSendOptions[] = "send_options";
+constexpr char kMaxRetries[] = "maxRetries";
+constexpr char kPreflightCommitment[] = "preflightCommitment";
+constexpr char kSkipPreflight[] = "skipPreflight";
+
+bool IsValidCommitmentString(const std::string& commitment) {
+  return commitment == "processed" || commitment == "confirmed" ||
+         commitment == "finalized";
+}
+
+}  // namespace
+
+SolanaTransaction::SendOptions::SendOptions() = default;
+SolanaTransaction::SendOptions::~SendOptions() = default;
+SolanaTransaction::SendOptions::SendOptions(
+    const SolanaTransaction::SendOptions&) = default;
+SolanaTransaction::SendOptions::SendOptions(
+    absl::optional<uint64_t> max_retries_param,
+    absl::optional<std::string> preflight_commitment_param,
+    absl::optional<bool> skip_preflight_param)
+    : max_retries(max_retries_param),
+      preflight_commitment(preflight_commitment_param),
+      skip_preflight(skip_preflight_param) {}
```

Comment:
These should be `std::move` not copy

Reply:
https://github.com/brave/brave-core/pull/13764/commits/01421e648f43c4347572392089a8063ed95d3a1f#diff-813b4a863ad7d6dd38df6f140065b560949a28b3b73cbb16fbd00c6aa5d8d04bR46-R48